CHANGES.txt

```diff
@@ -5,6 +5,17 @@ CHANGES for crate-dbal
 Unreleased
 ==========
 
+ - Updated crate-pdo to ``1.0.0`` which includes following changes:
+
+   - Added support for SSL via ``PDO::CRATE_ATTR_SSL_MODE``
+
+   - Updated provisioning using ubuntu xenial64 and the latest version of crate
+
+   - BREAKING: Refactored the internal http server/client implementation to a
```

Comment:
can we remove that? I don't get why an internal api change is a breaking change..

Reply:
Ok - this was taken from the crate-pdo change log.